pkg/metadata/labels.go

```diff
@@ -41,10 +41,17 @@ const (
 	InstanceLabel          = LabelKROPrefix + "instance-name"
 	InstanceNamespaceLabel = LabelKROPrefix + "instance-namespace"
 
+	// Parent Identifier Labels. These are set by the instance reconciler that creates/updates the resource.
 	ResourceGraphDefinitionIDLabel        = LabelKROPrefix + "resource-graph-definition-id"
 	ResourceGraphDefinitionNameLabel      = LabelKROPrefix + "resource-graph-definition-name"
 	ResourceGraphDefinitionNamespaceLabel = LabelKROPrefix + "resource-graph-definition-namespace"
 	ResourceGraphDefinitionVersionLabel   = LabelKROPrefix + "resource-graph-definition-version"
+
+	// Source Identifier Labels. These are set by the instance reconciler that reconciles the resource.
+	SourceResourceGraphDefinitionIDLabel        = LabelKROPrefix + "source-resource-graph-definition-id"
```

Comment:
since these labels are exposed to users we should probably document them a little bit better. they will likely be looked up by technically versed users, and maybe we should even think about explaining them in the docs (doesn't have to be in this PR imo)